src/components/CurrentEvent.js

```diff
@@ -53,6 +54,7 @@ export default function CurrentEvent({event, userData, userId}) {
       <div className="container">
         <div className="description-CE-container">
           <h5 className="description-price-title"><span className="price-title">Minimum Spend:</span><span className="price">${event.minimum_spend}</span></h5>
+          <h5 className="description-price-title"><span className="price-title">Maximum Spend:</span><span className="price">${event.minimum_spend + 30} </span></h5>
```

Comment:
Why are you adding a maximum spend here? We already decided not to do this.

Reply:
Because That's what made sense.